database/src/main/scala/cromwell/database/obj/BackendExecution.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.database.obj
+
+case class BackendExecution
+(
+  backendKVStoreID: Int,
```

Comment:
See my example on "Cars" over in Chris' PR. The auto-incremented PK goes last, and should be wrapped in a option with a default `None`.